lib/simulator-xcode-8.js

```diff
@@ -53,6 +53,29 @@ class SimulatorXcode8 extends SimulatorXcode7 {
     }
   }
 
+  async openUrl (url) {
+    const SAFARI_STARTUP_TIMEOUT = 15 * 1000;
+
+    if (!await this.isRunning()) {
+      throw new Error(`Tried to open ${url}, but Simulator is not in Booted state`);
+    }
+    const launchTimestamp = process.hrtime();
+    try {
+      await waitForCondition(async () => {
+        try {
+          await exec('bash', ['-c', `xcrun simctl launch ${this.udid} com.apple.mobilesafari ${url}`]);
```

Comment:
What is the reason for running this through `bash` instead of just running `xcrun` (like `await exec('xcrun', ['simctl', 'launch', this.udid, 'com.apple.mobilesafari', url]);`)?